File: robocode2-server/src/main/java/net/robocode2/model/events/BotDeathEvent.java
package net.robocode2.model.events;

import lombok.Value;

/**
 * Event sent when a bot has been killed.
 * 
 * @author Flemming N. Larsen
 */
@Value
public class BotDeathEvent implements Event {

	/** Bot id of the victim that got killed */
	int victimId;
}

File: robocode2-gui/src/main/kotlin/net/robocode2/gui/model/event/BotDeathEvent.kt
package net.robocode2.gui.model.event

import net.robocode2.gui.model.comm.Content
import net.robocode2.gui.model.comm.ContentType

class BotDeathEvent(
        val victimId: Int
) : Content(type = ContentType.BOT_DEATH_EVENT.type)
